### PR TITLE
Add Thoth reports references

### PR DESCRIPTION
**Pull request by pacospace**
Signed-off-by: Francesco Murdaca <fmurdaca@redhat.com>

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Add Thoth reports reference

### CODE REVIEW

File: thoth/slo_reporter/sli_references.py [hunk 1]
```diff
@@ -46,5 +46,12 @@ def _add_dashbords(start_time_epoch: datetime.datetime, end_time_epoch: datetime
     )
     html_inputs["Thoth SLI/SLO"] = sli_slo_dashboard_url
 
+    sli_slo_dashboard_name = f"thoth-reports"
+    sli_slo_dashboard_url = (
+        f"https://grafana.datahub.redhat.com/dashboard/db/{sli_slo_dashboard_name}?"
+        + f"refresh=1m&orgId=1&from={start_time_epoch}&to={end_time_epoch}"
+    )
+    html_inputs["Thoth Reports"] = sli_slo_dashboard_url
```
Comment (goern) [inline thoth/slo_reporter/sli_references.py:54]: what are those reports? and description we can provide so that people know why to click the link?

Comment (pacospace) [reply thoth/slo_reporter/sli_references.py]: Added description for each reference.

File: thoth/slo_reporter/sli_references.py [hunk 2]
```diff
@@ -37,14 +37,30 @@ def _add_dashbords(start_time_epoch: datetime.datetime, end_time_epoch: datetime
...
+    }
+
+    sli_slo_dashboard_name = f"thoth-reports"
+    sli_slo_dashboard_url = (
+        f"https://grafana.datahub.redhat.com/dashboard/db/{sli_slo_dashboard_name}?"
+        + f"refresh=1m&orgId=1&from={start_time_epoch}&to={end_time_epoch}"
+    )
+    html_inputs["Thoth Reports"] = {
+        "url": sli_slo_dashboard_url,
+        "description": "Dashboard for summary reports created."
```
Comment (goern) [inline thoth/slo_reporter/sli_references.py:62]: this is the dashboard of all slo reports created in the past?

Comment (pacospace) [reply thoth/slo_reporter/sli_references.py:62]: Yes all metrics we push every time we send a SLI report

**Comment by goern** [comment]
recheck

**Pull request merged**